MobileWallet/Screens/Home/HomeViewController.swift

```diff
@@ -367,16 +367,15 @@ class HomeViewController: UIViewController, FloatingPanelControllerDelegate, Tra
 
     // MARK: - Navigation
 
-    //TODO add back when working on next screen
-//    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-//        //TODO move segue identifiers to enum
-//        if let identifier = segue.identifier {
-//            if identifier == "HomeToTransactionDetails" {
-//                let transactionVC = segue.destination as! TransactionViewController
-//                transactionVC.transaction = selectedTransaction
-//            }
-//        }
-//    }
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        //TODO move segue identifiers to enum
+        if let identifier = segue.identifier {
+            if identifier == "HomeToTransactionDetails" {
+                let transactionVC = segue.destination as! TransactionViewController
```

Comment:
Do not use !. if let transactionVC = segue.destination as? TransactionViewController {

Reply:
Thanks, fixed 👍